react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -107,6 +108,12 @@ AutocompleteInput.propTypes = {
   iconBlockClass: PropTypes.string,
   /** Identify if the search input should autofocus or not */
   autoFocus: PropTypes.bool,
+  iconsProps: PropTypes.shape({
```

Comment:
🤔 should we create two separate props for each icon?

Reply:
I thought about it... but I think it's better to have a single prop for consistency.
If you set a viewBox for one of the icons, for example, you're setting it for the others as well.

Reply:
> If you set a viewBox for one of the icons, for example, you're setting it for the others as well
not sure if I get this.

is this behaviour unavoidable? If so, it of course makes sense for both icons to share the same prop, but I don't think this is the ideal behaviour.